src/components/BookForm.js

```diff
@@ -67,7 +67,7 @@ const BookForm = () => {
             <option value="comedy">Comedy</option>
             <option value="religious">Religious</option>
           </select>
-          <button type="submit">Add Book</button>
+          <button className="addBook" type="submit">Add Book</button>
         </div>
       </form>
     </div>
```

Comment:
- [ ] Please style the form to add books exactly like the zeplin design. Here are some things to consider in order to achieve the design:
1. Use the specified font-family and color
2. Remove the input labels. The placeholders are sufficient according to the design template
3. Style the submit form according to the design specifications taking note of the background-color, font-family, font-size and font-weight
4. Please add a horizontal border just above the "Add Book" form in order to replicate the design template 👍 
### Zeplin Design
![image](https://github.com/berylcl/bookstore/assets/46839548/142b0d33-e0c5-47a6-85d2-64b2dc343f07)

### Your Implementation
![image](https://github.com/berylcl/bookstore/assets/46839548/8b96e59f-c0bd-4c61-a89d-217c64663f6d)